lib/util.js

```diff
@@ -7,7 +7,7 @@ function * ensurePGStatStatement (db) {
   const query = `
 SELECT exists(
   SELECT 1 FROM pg_extension e LEFT JOIN pg_namespace n ON n.oid = e.extnamespace
-  WHERE e.extname='pg_stat_statements' AND n.nspname = 'public'
+  WHERE e.extname='pg_stat_statements' AND n.nspname IN ('public', 'heroku_ext')
```

Comment:
@iriberri does this check even need to be present? if the extension is in a different schema, what happens? will it fail to execute? can the user recover?